### ISSUE TITLE
[Legal] Unredistributable files in thirdparty/openblas

### ISSUE THREAD

**Issue by krop** [opentoonz/opentoonz#3928]
The openSUSE legal bot detected suspicious files when I submitted the opentoonz 1.5 update.

Looks like the faulty file was added 5 years ago. Better late than never :)

thirdparty/openblas/xianyi-OpenBLAS-e6e87a2 contains a lapack-3.4.1.tgz file. This archive contains files which are property of Intel and cannot be distributed freely.
From lapack-3.4.1/lapacke/example/example_DGESV_rowmajor.c and ZGESV_rowmajor.c contained in this archive:

```
*  Copyright (C) 2009-2011 Intel Corporation. All Rights Reserved.
*  The information and material ("Material") provided below is owned by Intel
*  Corporation or its suppliers or licensors, and title to such Material remains
*  with Intel Corporation or its suppliers or licensors. The Material contains
*  proprietary information of Intel or its suppliers and licensors. The Material
*  is protected by worldwide copyright laws and treaty provisions. No part of
*  the Material may be copied, reproduced, published, uploaded, posted,
*  transmitted, or distributed in any way without Intel's prior express written
*  permission. No license under any patent, copyright or other intellectual
*  property rights in the Material is granted to or conferred upon you, either
*  expressly, by implication, inducement, estoppel or otherwise. Any license
*  under such intellectual property rights must be express and approved by Intel
*  in writing.
```
tl;dr;
Even if opentoonz doesn't use the example files in the tarball contained in this thirdparty folder, redistributing this code must be allowed by Intel (and distributions are not allowed to ship these files).

**Comment by krop** [comment]
That's still an issue in opentoonz 1.6
